packages/itwinui-css/src/style/global.scss

```diff
@@ -3,7 +3,9 @@
 @import './index';
 @import './anchor';
 
-@layer itwinui-v1 {
+@layer itwinui-v1, itwinui.v1, itwinui.v2, itwinui.v3;
```

Comment:
Can the `itwinui-v1` layer be removed?

Reply:
no, we need both. https://github.com/iTwin/iTwinUI/pull/1412#discussion_r1263716496

users might have the old stylesheet included somewhere